modules/apigee_edge_apidocs/config/optional/views.view.apigee_edge_api_docs.yml

```diff
@@ -0,0 +1,289 @@
+langcode: en
+status: true
+dependencies:
+  config:
+  - system.menu.main
+  module:
+  - apigee_edge_apidocs
+  - text
+id: apigee_edge_api_docs
+label: 'Apigee Edge API Docs'
+module: views
+description: 'APIs published using the API Docs Apigee Edge module.'
+tag: ''
+base_table: apidoc_field_data
+base_field: id
+core: 8.x
+display:
+  default:
+    display_plugin: default
+    id: default
+    display_title: Master
+    position: 0
+    display_options:
+      access:
```

Comment:
I would have expected that this view checks for the "view published apidoc entities" permission.